components/Firebird/Deployments/ExistingDeployment/SelectedDeployment/StatusStream.tsx

```diff
@@ -0,0 +1,71 @@
+import classNames from "classnames"
+import { DeploymentStatus } from "@/utils/firebird/types"
+import { DeploymentEventType, EventStreamEvents } from "@/utils/firebird/types"
+import Loading from "@/components/Loading"
+import {
+  DeploymentStepFailedIcon,
+  DeploymentStepFinishedIcon,
+} from "@/utils/icons"
+
+export default function StatusStream({
+  status,
+  progress,
+}: {
+  status: DeploymentStatus | undefined
+  progress: Map<EventStreamEvents, DeploymentEventType>
+}) {
+  if (status === DeploymentStatus.ACTIVE) return null
+
+  const eventDisplayNames =
+    status === DeploymentStatus.STARTING
+      ? {
+          [EventStreamEvents.START_INSTANCE]: "Starting Instance",
+        }
+      : {
+          [EventStreamEvents.STOP_INSTANCE]: "Stopping Instance",
+        }
```

Comment:
Could you clean this up? Bit hard to read.

Something similar to how Martin organized here would be great: https://github.com/kwilteam/db-admin/blob/53d4bfe9caaad5e1a6c01ee8963b844f13b5a576/components/Firebird/Deployments/ExistingDeployment/SelectedDeployment/DeploymentStatusStream.tsx#L21